server/src/start-worker.ts

```diff
@@ -7,8 +7,8 @@ export async function startWorker() {
   try {
     const config = loadAppConfigFromEnv(process.env);
     const app = await bootstrap(config);
-    runDistributionWorker(app, config.distributionInterval * MILLISECONDS_PER_MINUTE);
-    runVettedBeneficiaryDistributionWorker(app, config.vettedDistributionInterval * MILLISECONDS_PER_MINUTE);
+    runDistributionWorker(app, `0 */${config.distributionInterval} * * * *`);
+    runVettedBeneficiaryDistributionWorker(app, `0 */${config.vettedDistributionInterval} * * * *`);
```

Comment:
`config.distributionInterval` should be passed as-is to the worker because `config.distributionInterval` itself will be set using the cron syntax.

Same with `config.vettedDistributionInterval`